avocado/utils/data_structures.py

```diff
@@ -52,6 +53,73 @@ def geometric_mean(values):
     return math.exp(sum([math.log(number) for number in values]) / no_values)
 
 
+def compare_matrices(matrix1, matrix2, treshold=0.05):
+    """
+    Compare 2 matrices nxm and return a matrix nxm with comparison data and
+    stats. When the first columns match, they are considered as header and
+    included in the results intact.
+
+    :param matrix1: Reference Matrix of floats; first column could be header.
+    :param matrix2: Matrix that will be compared; first column could be header
+    :param treshold: Any difference greater than this percent treshold will be
```

Comment:
Oh, I didn't noticed this earlier, but the correct spelling is `threshold`. Would you please update it?